packages/generator-electrode-component/app/index.js

```diff
@@ -136,6 +145,9 @@ var ReactComponentGenerator = yeoman.generators.Base.extend({
 	},
 
 	end: function() {
+		if (this.quoteType === "'") {
+			this.spawnCommandSync("node_modules/.bin/eslint", ["--fix", "src", "demo", "example", "test", "--ext", ".js,.jsx"]);
```

Comment:
Only kicker is that if there is a eslint issue with the generated code it will not remove the double quotes.